### PR TITLE
docs: Update deployment create help wording

### PR DESCRIPTION
**Pull request by karencfv**
**DO NOT MERGE until current release cycle has finalised and the following link works: https://elastic.co/guide/en/cloud/current/ec-api-deployment-crud.html#ec_create_a_deployment**

## Description
In preparations to the upcoming release, this PR updates the output of the `ecctl deployment create --help` command to include a link to our official documentation.

## Related Issues
Related: https://github.com/elastic/ecctl/issues/161

## How Has This Been Tested?
By running `make docs`

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improves code quality but has no user-facing effect)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

### CODE REVIEW

File: cmd/deployment/create_help.go [hunk 1]
```diff
@@ -79,8 +79,8 @@ $ ecctl deployment create --request-id=GMZPMRrcMYqHdmxjIQkHbdjnhPIeBElcwrHwzVlhG
 	// Remove temporary constants when reads for deployment templates are available on ESS
 	createLongTemp = `Creates a deployment which is defined from a file definition using the --file=<file path> (shorthand: -f) flag.
 
-//You can create a definition by using the sample JSON seen here:
-//  https://elastic.co/guide/en/cloud/current/ec-api-deployment-crud.html#ec_create_a_deployment`
+You can create a definition by using the sample JSON seen here:
```
Comment (alaudazzi) [inline cmd/deployment/create_help.go:82]:
```suggestion
You can create a definition by using the sample JSON:
```

File: docs/ecctl_deployment_create.adoc [hunk 2]
```diff
@@ -8,8 +8,8 @@ Creates a deployment
 
 Creates a deployment which is defined from a file definition using the --file=+++<file path="">+++(shorthand: -f) flag.+++</file>+++
 
-//You can create a definition by using the sample JSON seen here:
-//  https://elastic.co/guide/en/cloud/current/ec-api-deployment-crud.html#ec_create_a_deployment
+You can create a definition by using the sample JSON seen here:
```
Comment (alaudazzi) [inline docs/ecctl_deployment_create.adoc:11]:
```suggestion
You can create a definition by using the sample JSON:
```

File: docs/ecctl_deployment_create.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ Creates a deployment
 
 Creates a deployment which is defined from a file definition using the --file=<file path> (shorthand: -f) flag.
 
-//You can create a definition by using the sample JSON seen here:
-//  https://elastic.co/guide/en/cloud/current/ec-api-deployment-crud.html#ec_create_a_deployment
+You can create a definition by using the sample JSON seen here:
```
Comment (alaudazzi) [inline docs/ecctl_deployment_create.md:9]:
```suggestion
You can create a definition by using the sample JSON:
```

**Comment by nrichers** [comment]
@karencfv One other thing i didn't mention in my previous comment: this PR also needs to be cherry picked into the `v1.0.0-beta2` branch, since that is where the default public docs are being built from.

**Comment by karencfv** [comment]
Thanks @nrichers and @alaudazzi! I have amended the text. 

Once this is merged, we will immediately release ecctl v1.0.0-beta3. Maybe it will make sense to cut a new branch and point the docs build to that branch?

**Comment by nrichers** [comment]
> Thanks @nrichers and @alaudazzi! I have amended the text.
> 
> Once this is merged, we will immediately release ecctl v1.0.0-beta3. Maybe it will make sense to cut a new branch and point the docs build to that branch?

@karencfv that's easy enough to do. After you have created the `v1.0.0-beta3` branch

* Change `current` to point to that branch in https://github.com/elastic/docs/blob/master/conf.yaml#L836
* Add the  `v1.0.0-beta3` branch to https://github.com/elastic/docs/blob/master/conf.yaml#L837
* Do a full doc rebuild to make beta3 the default docs at https://elasticsearch-ci.elastic.co/job/elastic+docs+master+build/build?delay=0sec

  image

Happy to help with these steps, just ping me on Slack!

**Comment by karencfv** [comment]
@nrichers Thanks for the detailed instructions on how to update the branch. They seem pretty straightforward. I'll update once we have created the release!

**Pull request merged**